File: Stefan Alexandru/Day 17/problem.js
// https://www.codewars.com/kata/build-a-pile-of-cubes/train/javascript
function findNb(m) {
    for (let n = 1; ; n++) {
        if (m > 0) {
            m = m - Math.pow(n, 3);
        } else if (m === 0) {
            return n;
        } else if (m < 0) {
            return -1;
        }
    }
}

File: Stefan Alexandru/Day 17/problem.py
# https://www.codewars.com/kata/build-a-pile-of-cubes/train/python
def find_nb(m):
    for value in range(1, m):
        m -= value ** 3
        if m == 0:
            return value
        elif m < 0:
            return -1
